src/lib/style/components/_error.scss

```diff
@@ -1,6 +1,9 @@
 .Error {
-  div {
+  div.alert {
     text-align: center;
+    display: flex;
+    justify-content: space-around;
```

Comment:
Might look better with `space-between` but I don't know if I want the Reload button hidden so far off the screen in the table component.